app/models/user.rb

```diff
@@ -249,8 +250,9 @@ def set_remember_fields
   end
 
   def calculate_score
-    score = (articles.where(featured: true).size * 100) + comments.sum(:score)
-    update_column(:score, score)
+    user_reaction_points = Reaction.user_vomits.where(reactable_id: id).sum(:points)
+    calculated_score = (badge_achievements_count * 10) + user_reaction_points
+    update_column(:score, calculated_score)
```

Comment:
Just leaving a note here that we might potentially need to index this column for optimization.

Reply:
Great call. My theme of this PR in general is the acknowledgment that we literally only use this in _one place_ and that is the check if it's less than 0... But as a factor we could rank/query in the future and most likely will want to build on top of this, that makes sense.

Seems out of scope here, but good call. cc: @citizen428